packages/pages/src/common/src/template/client.ts

```diff
@@ -59,10 +62,9 @@ const generateAndSaveClientHydrationTemplates = async (path: string) => {
  */
 const getClientPath = (templatePath: string): string => {
   const parsedPath = path.parse(templatePath);
-  return path.join(
-    parsedPath.dir,
-    parsedPath.name + ".client" + parsedPath.ext
-  );
+  parsedPath.name = parsedPath.name + ".client";
+  parsedPath.base = parsedPath.name + parsedPath.ext;
```

Comment:
If you change name do you need to change base for format to work?

Reply:
Yup